tsconfig.json

```diff
@@ -13,7 +13,10 @@
       "#polyfills/*": ["./src/polyfills/*"],
       "#preact/*": ["./src/preact/*"],
       "#purifier/*": ["./src/purifier/*"],
-      "#service/*": ["./src/service/*"]
+      "#service/*": ["./src/service/*"],
+
+      "#test/*": ["./test/*"],
+      "#testing/*": ["./testing/*"]
```

Comment:
Since this is a `tsconfig` now, do you need to mark `allowJS: true`?

Reply:
Added, thank you (in the process of installing VSCode so I can test these changes myself)